index.d.ts

```diff
@@ -74,6 +75,7 @@ declare global {
   // @ts-ignore
   var fetch: typeof _fetch
   var fs: typeof _fs
+  var globby: typeof _globby.globby & typeof _globby
```

Comment:
‘globby’ is exported as index const, right? so it should also be referenced as ‘export const globby: …’ a few lines above (same as ‘argv’)

Reply:
Ok.